ledger/ledger.go

```diff
@@ -128,3 +137,21 @@ func (c *Client) CloseChannel(ctx context.Context, signedChannelState *ledgerpb.
 		return err
 	})
 }
+
+func NewPublicKey(privK ic.PrivKey, pubK ic.PubKey) (*ledgerpb.SignedPublicKey, error) {
```

Comment:
Suggest to change the name to make it more understandable.

Reply:
NewSignedPublicKey?

Reply:
done the name change.